golang/testsuite/networks_impl/test_network.go

```diff
@@ -61,13 +73,16 @@ func (network *TestNetwork) SetupDatastoreAndTwoApis() error {
 		return stacktrace.NewError("Cannot add API services to network; one or more API services already exists")
 	}
 
-	configFactory := datastore.NewDatastoreContainerConfigFactory(network.datastoreServiceImage)
-	datastoreServiceContext, hostPortBindings, err := network.networkCtx.AddService(datastoreServiceId, configFactory)
+	datastoreContainerCreationConfig := getDataStoreContainerCreationConfig()
```

Comment:
If you want to make this even tighter, you could have a single function that returns both of these - e.g. `getDatatstoreContainerConfigs`